### PR TITLE
move test-chuck and sort dev deps

### PR DESCRIPTION
**Pull request by escherize**
I should've made this a dev dependency, since it's not something we will likely use in the actual product

### CODE REVIEW

File: deps.edn [hunk 1]
```diff
@@ -193,23 +192,26 @@
...
     reifyhealth/specmonstah      {:mvn/version "2.0.0"
                                   :exclusions  [org.clojure/clojure
-                                                org.clojure/clojurescript]}
-    ring/ring-mock               {:mvn/version "0.4.0"}
-    talltale/talltale            {:mvn/version "0.5.8"}
-    lambdaisland/deep-diff2      {:mvn/version "2.3.127"}}
+                                                org.clojure/clojurescript]} ; lets you write test fixtures that are clear, concise, and easy to maintain (clojure.spec)
+    ring/ring-mock               {:mvn/version "0.4.0"}                     ; creating Ring request maps for testing purposes
+    talltale/talltale            {:mvn/version "0.5.8"}                     ; generates fake data, useful for prototyping or load testing
+    }
```
Comment (escherize) [inline deps.edn:215]: I sorted these, and added explanations to many of them.

File: deps.edn [hunk 2]
```diff
@@ -193,23 +192,26 @@
   ;;    (dev/start!)
   :dev
   {:extra-deps
-   {clj-http-fake/clj-http-fake  {:mvn/version "1.0.3"
+   {
```
Comment (tsmacdonald) [inline deps.edn:195]: for the main deps above we have the `{` on the same line as the first dependency

File: deps.edn [hunk 3]
```diff
@@ -193,23 +192,26 @@
...
+    clj-http-fake/clj-http-fake  {:mvn/version "1.0.3"
                                   :exclusions  [slingshot/slingshot]}
-    clj-kondo/clj-kondo          {:mvn/version "2022.08.03"}                    ; this is not for RUNNING kondo, but so we can hack on custom hooks code from the REPL.
+    clj-kondo/clj-kondo          {:mvn/version "2022.08.03"}                ; this is not for RUNNING kondo, but so we can hack on custom hooks code from the REPL.
     cloverage/cloverage          {:mvn/version "1.2.2"}
-    djblue/portal                {:mvn/version "0.30.0"}
+    com.gfredericks/test.chuck   {:mvn/version "0.2.13"}                    ; generating strings from regexes (useful with malli)
+    djblue/portal                {:mvn/version "0.30.0"}                    ; ui for inspectinv values
```
Comment (tsmacdonald) [inline deps.edn:201]:
```suggestion
    djblue/portal                {:mvn/version "0.30.0"}                    ; ui for inspecting values
```

File: deps.edn [hunk 4]
```diff
@@ -193,23 +192,26 @@
...
     eftest/eftest                {:mvn/version "0.5.9"}
     io.github.camsaul/humane-are {:mvn/version "1.0.2"}
-    jonase/eastwood              {:mvn/version "1.2.2"}
-    methodical/methodical        {:mvn/version "0.15.0.1"}
+    jonase/eastwood              {:mvn/version "1.2.2"}                     ; inspects namespaces and reports possible problems using tools.analyzer
+    lambdaisland/deep-diff2      {:mvn/version "2.3.127"}                   ; way better diffs
```
Comment (tsmacdonald) [inline deps.edn:205]: TIL; where/how is this used?

**Review by tsmacdonald — APPROVED** [review]
few nitpicks I'll let you deal with on your own, but otherwise LGTM

File: deps.edn (hunk 4 above)
Comment (tsmacdonald) [reply deps.edn:205]: aha https://github.com/metabase/metabase/blob/d18e3da72b346124ddfd320457ac7d1b1ab5ceb7/dev/src/dev/debug_qp.clj#L173-L194

**Review by camsaul — APPROVED** [review]
LGTM. I could go either way on the `{` so you guys can figure that out

**Review by tsmacdonald — APPROVED** [review]

**Pull request merged**